superset-frontend/src/SqlLab/components/TabbedSqlEditors/index.jsx

```diff
@@ -401,6 +409,19 @@ class TabbedSqlEditors extends React.PureComponent {
       );
     });
 
+    const emptyTab = (
+      <EditableTabs.TabPane
+        key={Math.random()}
+        data-key={Math.random()}
```

Comment:
can we just set this to `0`?

Reply:
Yeah I was worried about hardcoding a number, but I think that you are right and that this would be the ebst way forward.